cylc/uiserver/schema.py

```diff
@@ -298,6 +281,19 @@ async def resolve_logs(
         ):
             yield item
 
+    logs = graphene.Field(
```

Comment:
The reason passing the `workflowID` through didn't work was because this needed to be a `Field`. No idea why but that's how the cylc-flow subs worked so I copied the approach.